plenum/server/node.py

```diff
@@ -2071,12 +2069,12 @@ def processRequest(self, request: Request, frm: str):
             return
 
         # If the node is not already processing the request
-        if not self.isProcessingReq(*request.key):
-            self.startedProcessingReq(*request.key, frm)
+        if not self.isProcessingReq((request.identifier, request.reqId)):
+            self.startedProcessingReq((request.identifier, request.reqId), frm)
```

Comment:
Although we may keep did+reqId to be a key in `requestSender`, I think eventually we will come to replacing it by `digest`...
For example, we may have two clients (with different `frm` keys) sending requests with the same did+reqId. According to the current logic, Replies will be sent just to one of the clients.

Reply:
Changed a key in request Sender to digest.